easybuild/easyconfigs/h/hatchling/hatchling-1.27.0-GCCcore-13.3.0.eb

```
@@ -0,0 +1,56 @@
+easyblock = 'PythonBundle'
+
+name = 'hatchling'
+version = '1.27.0'
+
+homepage = 'https://hatch.pypa.io'
+description = """Extensible, standards compliant build backend used by Hatch,
+a modern, extensible Python project manager."""
+
+toolchain = {'name': 'GCCcore', 'version': '13.3.0'}
+
+builddependencies = [
+    ('binutils', '2.42'),
+]
```

```suggestion
    ('setuptools', '80.9.0'),
]
```

When installing the `hatchling` extension.
```
ModuleNotFoundError: No module named 'packaging.licenses'
```